.github/workflows/release.yaml

```diff
@@ -65,21 +65,11 @@ jobs:
           gsutil -h "Cache-Control:no-cache,max-age=0" cp build/helm/${version}.tgz gs://helm.coder.com/logstream-kube
           gsutil -h "Cache-Control:no-cache,max-age=0" cp build/helm/index.yaml gs://helm.coder.com/logstream-kube
 
-      - name: Create Release
-        uses: actions/create-release@v1
-        id: create_release
-        env:
-          GITHUB_TOKEN: ${{ github.token }}
-        with:
-          release_name: ${{ steps.version.outputs.version }}
-          tag_name: ${{ github.ref }}
-
-      - name: Upload Helm Release Asset
-        uses: actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN: ${{ github.token }}
+      - name: Create and upload release
+        uses: softprops/action-gh-release@v2
```

Comment:
Are we using this action anywhere else?
Would it make more sense to simply use `gh release create`?

Reply:
Yes.
There is one issue, though: if we create a release manually on github.com, which also creates a tag, we need to somehow skip this step. Otherwise, it fails with an error that the release already exists.
Something like [this](https://github.com/coder/coder-logstream-kube/actions/runs/10160128067/job/28095813641#step:10:20):
```console
Error: Validation Failed: {"resource":"Release","code":"already_exists","field":"tag_name"}
```

Reply:
Would this serve our purpose here?

> Use `--verify-tag` to abort the release if the tag doesn't already exist.
https://cli.github.com/manual/gh_release_create

Reply:
refactored to use `gh release create`